instrumentation/github.com/tidwall/buntdb/splunkbuntdb/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/signalfx/splunk-otel-go/instrumentation/github.com/tidwall/buntdb/splunkbuntdb
+
+go 1.15
+
+require (
+	github.com/signalfx/splunk-otel-go v0.6.0
+	github.com/tidwall/buntdb v1.2.7
+	go.opentelemetry.io/otel v1.2.0
+	go.opentelemetry.io/otel/trace v1.2.0
+)
+
+// replace github.com/signalfx/signalfx-go-tracing => ../../../
```

Comment:
why is it commented?

Reply:
Agreed. Please uncomment and fix the relative path.